doc/plugins/response-rewrite-cn.md

```diff
@@ -0,0 +1,79 @@
+<!--
+#
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+-->
+
+[English](response-rewrite.md)
+# response-rewrite
+
+该插件支持修改上游服务返回的body和header信息。
+可以设置 `Access-Control-Allow-*` 等header信息，来实现 CORS (跨域资源共享)的功能。
+
+#### 配置参数
+|名字    |可选|说明|
+|------- |-----|------|
+|body          |可选| 修改上游返回的 `body` 内容，如果设置了新内容，header里面的content-type字段也会被修改|
```

Comment:
It is also useful to support the specified response code.

Reply:
will add in the next commit